The job **`mingw-check-tidy`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/9587317731/job/26437098694) [(plain)](https://github.com/rust-lang/rust/commit/7fc117589e0f8850d4f166c5f80d874e89e60561/checks/26437098694/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
Getting action download info
Download action repository 'msys2/setup-msys2@v2.22.0' (SHA:cc11e9188b693c2b100158c3322424c4cc1dadea)
Download action repository 'actions/checkout@v4' (SHA:692973e3d937129bcbf40652eb9f2f61becf3332)
Download action repository 'actions/upload-artifact@v4' (SHA:65462800fd760344b1a7b4382951275a0abb4808)
Complete job name: PR - mingw-check-tidy
git config --global core.autocrlf false
shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
---
COPY scripts/sccache.sh /scripts/
RUN sh /scripts/sccache.sh

COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt \
    && pip3 install virtualenv
COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
COPY host-x86_64/mingw-check/validate-error-codes.sh /scripts/

# NOTE: intentionally uses python2 for x.py so we can test it still works.
# NOTE: intentionally uses python2 for x.py so we can test it still works.
# validate-toolstate only runs in our CI, so it's ok for it to only support python3.
ENV SCRIPT TIDY_PRINT_DIFF=1 python2.7 ../x.py test \
           --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
# This file is autogenerated by pip-compile with Python 3.10
# by the following command:
#
#    pip-compile --allow-unsafe --generate-hashes reuse-requirements.in
---

#12 [5/8] COPY host-x86_64/mingw-check/reuse-requirements.txt /tmp/
#12 DONE 0.0s

#13 [6/8] RUN pip3 install --no-deps --no-cache-dir --require-hashes -r /tmp/reuse-requirements.txt     && pip3 install virtualenv
#13 0.411   Downloading binaryornot-0.4.4-py2.py3-none-any.whl (9.0 kB)
#13 0.423 Collecting boolean-py==4.0
#13 0.428   Downloading boolean.py-4.0-py3-none-any.whl (25 kB)
#13 0.439 Collecting chardet==5.1.0
---
#13 3.508 Building wheels for collected packages: reuse
#13 3.509   Building wheel for reuse (pyproject.toml): started
#13 3.844   Building wheel for reuse (pyproject.toml): finished with status 'done'
#13 3.845   Created wheel for reuse: filename=reuse-1.1.0-cp310-cp310-manylinux_2_35_x86_64.whl size=181117 sha256=f5f58750481f69515c2c0d1d503daf565e2565c370d07fc6aeb95fe3498b4269
#13 3.846   Stored in directory: /tmp/pip-ephem-wheel-cache-t4wj8jop/wheels/c2/3c/b9/1120c2ab4bd82694f7e6f0537dc5b9a085c13e2c69a8d0c76d
#13 3.848 Installing collected packages: boolean-py, binaryornot, setuptools, reuse, python-debian, markupsafe, license-expression, jinja2, chardet
#13 3.870   Attempting uninstall: setuptools
#13 3.871     Found existing installation: setuptools 59.6.0
#13 3.872     Not uninstalling setuptools at /usr/lib/python3/dist-packages, outside environment /usr
---
#13 5.081   Downloading virtualenv-20.26.2-py3-none-any.whl (3.9 MB)
#13 5.172      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 3.9/3.9 MB 44.2 MB/s eta 0:00:00
#13 5.225 Collecting filelock<4,>=3.12.2
#13 5.229   Downloading filelock-3.15.3-py3-none-any.whl (16 kB)
#13 5.257 Collecting platformdirs<5,>=3.9.1
#13 5.261   Downloading platformdirs-4.2.2-py3-none-any.whl (18 kB)
#13 5.277 Collecting distlib<1,>=0.3.7
#13 5.281   Downloading distlib-0.3.8-py2.py3-none-any.whl (468 kB)
#13 5.288      ━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━━ 468.9/468.9 KB 99.9 MB/s eta 0:00:00
#13 5.376 Installing collected packages: distlib, platformdirs, filelock, virtualenv
#13 5.539 Successfully installed distlib-0.3.8 filelock-3.15.3 platformdirs-4.2.2 virtualenv-20.26.2
#13 DONE 5.6s

#14 [7/8] COPY host-x86_64/mingw-check/validate-toolstate.sh /scripts/
#14 DONE 0.0s
---
DirectMap4k:      184256 kB
DirectMap2M:     8204288 kB
DirectMap1G:    10485760 kB
##[endgroup]
Executing TIDY_PRINT_DIFF=1 python2.7 ../x.py test            --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
+ TIDY_PRINT_DIFF=1 python2.7 ../x.py test --stage 0 src/tools/tidy tidyselftest --extra-checks=py:lint
    Finished `dev` profile [unoptimized] target(s) in 0.04s
##[endgroup]
downloading https://ci-artifacts.rust-lang.org/rustc-builds-alt/5c8459f1eceba84dff8d622768dae556ea7c2495/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz
extracting /checkout/obj/build/cache/llvm-5c8459f1eceba84dff8d622768dae556ea7c2495-true/rust-dev-nightly-x86_64-unknown-linux-gnu.tar.xz to /checkout/obj/build/x86_64-unknown-linux-gnu/ci-llvm
---
   Compiling tidy v0.1.0 (/checkout/src/tools/tidy)
    Finished `release` profile [optimized] target(s) in 27.18s
##[endgroup]
fmt check
##[error]Diff in /checkout/tests/run-make/inaccessible-temp-dir/rmake.rs at line 19:
 // Reason: `set_readonly` has no effect on directories
 
-use run_make_support::{fs_wrapper, rustc, test_while_readonly, target};
-use run_make_support::{fs_wrapper, rustc, test_while_readonly, target};
+use run_make_support::{fs_wrapper, rustc, target, test_while_readonly};
 fn main() {
     // Create an inaccessible directory.
     // Create an inaccessible directory.
fmt error: Running `"/checkout/obj/build/x86_64-unknown-linux-gnu/rustfmt/bin/rustfmt" "--config-path" "/checkout" "--edition" "2021" "--unstable-features" "--skip-children" "--check" "/checkout/tests/run-make/raw-dylib-link-ordinal/lib.rs" "/checkout/tests/run-make/volatile-intrinsics/main.rs" "/checkout/tests/run-make/dep-info-spaces/foo foo.rs" "/checkout/tests/run-make/dep-info-spaces/lib.rs" "/checkout/tests/run-make/dep-info-spaces/bar.rs" "/checkout/tests/run-make/raw-dylib-c/driver.rs" "/checkout/tests/run-make/raw-dylib-c/lib.rs" "/checkout/tests/run-make/cdylib/foo.rs" "/checkout/tests/run-make/cdylib/rmake.rs" "/checkout/tests/run-make/cdylib/bar.rs" "/checkout/tests/run-make/mixing-deps/rmake.rs" "/checkout/tests/run-make/mixing-deps/prog.rs" "/checkout/tests/run-make/mixing-deps/both.rs" "/checkout/tests/run-make/mixing-deps/dylib.rs" "/checkout/tests/run-make/used-cdylib-macos/dylib_used.rs" "/checkout/tests/run-make/issue-83112-incr-test-moved-file/main.rs" "/checkout/tests/run-make/extern-fn-reachable/dylib.rs" "/checkout/tests/run-make/mixing-libs/prog.rs" "/checkout/tests/run-make/mixing-libs/rlib.rs" "/checkout/tests/run-make/mixing-libs/dylib.rs" "/checkout/tests/run-make/link-path-order/main.rs" "/checkout/tests/run-make/reproducible-build/linker.rs" "/checkout/tests/run-make/reproducible-build/reproducible-build-aux.rs" "/checkout/tests/run-make/reproducible-build/reproducible-build.rs" "/checkout/tests/run-make/c-link-to-rust-staticlib/foo.rs" "/checkout/tests/run-make/c-link-to-rust-staticlib/rmake.rs" "/checkout/tests/run-make/wasm-custom-section/foo.rs" "/checkout/tests/run-make/wasm-custom-section/rmake.rs" "/checkout/tests/run-make/wasm-custom-section/bar.rs" "/checkout/tests/run-make/manual-link/foo.rs" "/checkout/tests/run-make/manual-link/main.rs" "/checkout/tests/run-make/rlib-format-packed-bundled-libs-2/main.rs" "/checkout/tests/run-make/rlib-format-packed-bundled-libs-2/native_dep.rs" "/checkout/tests/run-make/rlib-format-packed-bundled-libs-2/rust_dep.rs" "/checkout/tests/run-make/relocation-model/foo.rs" "/checkout/tests/run-make/reproducible-build-2/linker.rs" "/checkout/tests/run-make/reproducible-build-2/reproducible-build-aux.rs" "/checkout/tests/run-make/reproducible-build-2/reproducible-build.rs" "/checkout/tests/run-make/rlib-format-packed-bundled-libs-3/rust_dep_cfg.rs" "/checkout/tests/run-make/rlib-format-packed-bundled-libs-3/main.rs" "/checkout/tests/run-make/rlib-format-packed-bundled-libs-3/rust_dep.rs" "/checkout/tests/run-make/extern-fn-with-union/test.rs" "/checkout/tests/run-make/extern-fn-with-union/testcrate.rs" "/checkout/tests/run-make/rust-lld-custom-target/rmake.rs" "/checkout/tests/run-make/rust-lld-custom-target/lib.rs" "/checkout/tests/run-make/inaccessible-temp-dir/rmake.rs" "/checkout/tests/run-make/inaccessible-temp-dir/program.rs" "/checkout/tests/run-make/extern-fn-struct-passing-abi/test.rs" "/checkout/tests/run-make/output-with-hyphens/foo-bar.rs" "/checkout/tests/run-make/output-with-hyphens/rmake.rs" "/checkout/tests/run-make/mixing-formats/foo.rs" "/checkout/tests/run-make/mixing-formats/bar1.rs" "/checkout/tests/run-make/mixing-formats/rmake.rs" "/checkout/tests/run-make/mixing-formats/bar2.rs" "/checkout/tests/run-make/mixing-formats/baz.rs" "/checkout/tests/run-make/mixing-formats/baz2.rs" "/checkout/tests/run-make/no-cdylib-as-rdylib/foo.rs" "/checkout/tests/run-make/no-cdylib-as-rdylib/rmake.rs" "/checkout/tests/run-make/no-cdylib-as-rdylib/bar.rs" "/checkout/tests/run-make/rlib-chain/m4.rs" "/checkout/tests/run-make/rlib-chain/m1.rs" "/checkout/tests/run-make/rlib-chain/m2.rs" "/checkout/tests/run-make/rlib-chain/m3.rs" "/checkout/tests/run-make/raw-dylib-link-ordinal/driver.rs"` failed.
If you're running `tidy`, try again with `--bless`. Or, if you just want to format code, run `./x.py fmt` instead.
  local time: Wed Jun 19 19:40:18 UTC 2024
  network time: Wed, 19 Jun 2024 19:40:19 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/tests/run-make/inaccessible-temp-dir/rmake.rs
+++ b/tests/run-make/inaccessible-temp-dir/rmake.rs
@@ -19,7 +19,7 @@
 // Reason: `set_readonly` has no effect on directories
 // and does not prevent modification.
 
-use run_make_support::{fs_wrapper, rustc, test_while_readonly};
+use run_make_support::{fs_wrapper, rustc, target, test_while_readonly};
 
 fn main() {
     // Create an inaccessible directory.
@@ -28,6 +28,7 @@ fn main() {
         // Run rustc with `-Z temps-dir` set to a directory *inside* the inaccessible one,
         // so that it can't create `tmp`.
         rustc()
+            .target(&target())
             .input("program.rs")
             .arg("-Ztemps-dir=inaccessible/tmp")
             .run_fail()

